src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+C4DEXE="$1"
```

Comment:
I think we'll have to add `set -euo pipefail` to help propagate any failures in this script (ie. from C4D itself) to the session action.

Reply:
Good catch